packages/terra-data-grid/src/subcomponents/ColumnResizeHandle.jsx

```diff
@@ -1,80 +1,110 @@
 import React, {
-  useContext, useRef, useCallback, useState,
+  useContext, useRef, useEffect, useState,
 } from 'react';
 import PropTypes from 'prop-types';
 import { injectIntl } from 'react-intl';
+import * as KeyCode from 'keycode-js';
 import classNames from 'classnames/bind';
 import ThemeContext from 'terra-theme-context';
+import ColumnContext from '../utils/ColumnContext';
+import '../_elementPolyfill';
```

Comment:
Question about this: What part of the code is using this polyfill?

Reply:
Looks like we aren't so removed it here: 9340a4ceec311a633eef24d9bca8e6a00c211c75